src/ducks/SANCharts/Chart/axes.js

```diff
@@ -0,0 +1,14 @@
+import { drawAxes, drawAxesTicks } from '@santiment-network/chart/axes'
+import { getDateDayMonthYear } from './utils'
+
+const axesTickFormatters = {
+  datetime: getDateDayMonthYear
+}
+
+export function plotAxes (chart) {
+  const { ctx } = chart
+  drawAxes(chart)
+  ctx.fillStyle = '#9faac4'
```

Comment:
Is it a color from our design system? @DmitriVanGuard

Reply:
https://storybook.stage.internal.santiment.net/?selectedKind=Colors&selectedStory=List%20of%20all%20colors&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel